specification/postgresql/resource-manager/readme.md

```diff
@@ -28,13 +28,184 @@ These are the global settings for the PostgreSQL API.
 title: PostgreSQLManagementClient
 description: The Azure Database for PostgreSQL management API provides create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall rules, network configuration, security alert policies, log files and configurations with new business model.
 openapi-type: arm
-tag: package-flexibleserver-2025-06-01-preview
+tag: package-flexibleserver-2025-08-01
 ```
 
 ``` yaml $(package-singleservers)
 tag: package-2020-01-01
 ```
 
+### Tag: package-flexibleserver-2025-08-01
+
+These settings apply only when `--tag=package-flexibleserver-2025-08-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-flexibleserver-2025-08-01'
+input-file:
+  - Microsoft.DBforPostgreSQL/preview/2025-08-01/AdministratorsMicrosoftEntra.json
```

Comment:
Fix all the files paths to say stable not preview, will fix swagger avocado error